src/tree/SubscriptionTreeItem.ts

```diff
@@ -72,13 +75,21 @@ export class SubscriptionTreeItem extends SubscriptionTreeItemBase {
         wizardContext.repoHtmlUrl = await tryGetRemote(wizardContext);
         wizardContext.telemetry.properties.gotRemote = String(!!wizardContext.repoHtmlUrl);
 
+        wizardContext.fsPath = getSingleRootFsPath();
+
         await wizard.prompt();
         const newStaticWebAppName: string = nonNullProp(wizardContext, 'newStaticWebAppName');
         wizardContext.newResourceGroupName = newStaticWebAppName;
 
         await wizard.execute();
         context.showCreatingTreeItem(newStaticWebAppName);
 
+        if (wizardContext.fsPath) {
+            await updateWorkspaceSetting(appSubpathSetting, wizardContext.appLocation, wizardContext.fsPath);
```

Comment:
I'm a little unsure about the experience here. Seems like we can't be sure the current workspace matches up with the repo they're going to use for the static web app. Also would be annoying if I already had something set and you always overwrote the value. A few ideas:

1. Only update settings if you 'gotRemote'
1. Only update settings if I don't already have something set
1. Never update settings automatically and prompt to save defaults

Reply:
> Seems like we can't be sure the current workspace matches up with the repo they're going to use for the static web app
Yeah, true that.  I think idea 1. would help mitigate that at least.

>Also would be annoying if I already had something set and you always overwrote the value.
I thought about that, but I figured if someone went out of their way to change the value that was previously their default, then it probably wasn't the default that they needed.

Reply:
Sounds good